sbin/run.py

```diff
@@ -50,14 +50,25 @@
     f.write(json.dumps(schedule, ensure_ascii=False, indent=True))
 
 # launch scheduler
-tasks = subprocess.check_output(['ps', 'xw']).decode('utf8')
-sublime_is_running = "Sublime" in tasks or "sublime_text" in tasks
+def sublime_is_running():
+    PIPE = subprocess.PIPE
+    out, err = subprocess.Popen(['pgrep','[s|S]ubl'], stdout=PIPE, stderr=PIPE).communicate()
+    return not err and len(out) > 0
 
-if sublime_is_running:
-    subprocess.Popen(["subl", "-b", "--command", "unit_testing_run_scheduler"])
-else:
+if not sublime_is_running():
     subprocess.Popen(["subl"])
```

Comment:
If Sublime Text is not running, it is sufficient to launch `subl` without `unit_testing_run_scheduler`. Sublime Text will automatically launch the [scheduler ](https://github.com/randy3k/UnitTesting/blob/master/ut.py#L20) when it is lanuched.

The command `unit_testing_run_scheduler` is only relevent if Sublime Text is already opened, and a developer wants to run the tests from command line.